### ISSUE TITLE
com.novoda.gradle.release.AndroidLibrary$LibraryUsage.getGlobalExcludes()Ljava/util/Set

### ISSUE THREAD

**Issue by kkmike999** [novoda/bintray-release#216]
```
$ gradle clean build :library:bintrayUpload -PbintrayUser=xxxx -PbintrayKey=xxxx -PdryRun=false -x test

FAILURE: Build failed with an exception.

* What went wrong:
com.novoda.gradle.release.AndroidLibrary$LibraryUsage.getGlobalExcludes()Ljava/util/Set;

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
See https://docs.gradle.org/4.8.1/userguide/command_line_interface.html#sec:command_line_warnings
```

How to fix this?

**Comment by passsy** [comment]
which android gradle plugin version and gradle version are you using?

**Comment by kkmike999** [comment]
@passsy 3.1.3. i find source code `com.novoda.gradle.release.AndroidLibrary.LibraryUsage`，doesn't implement all methods from `UsageContext`, like `getGlobalExcludes` and so on.

**Comment by StefMa** [comment]
Duplicate of #177

**Comment by StefMa** [comment]
You have to use Gradle 4.4 or lower

**Comment by kkmike999** [comment]
@StephWerner i use gradle 4.4, but still has this problem

**Comment by StefMa** [comment]
Try a lower gradle version like 4.3 🙃

**Comment by kkmike999** [comment]
@StefMa android studio 3.1.3, gradle tools 3.1.3, doesn't support gradle version 4.4 lower T_T

**Comment by StefMa** [comment]
Try this one:
https://github.com/StefMa/bintray-release/

**Comment by 4ntoine** [comment]
I have the same issue with android library. Any solution in official plugin (here)?

**Comment by tasomaniac** [comment]
Please check #177 for more details. This should be fixed with version 0.9

**Comment by 4ntoine** [comment]
I can confirm 0.9 works for me, thanks.
